docs/how_to/query.md

```diff
@@ -85,6 +85,18 @@ An example of a configuration file with the ID generation scheme can be found [h
 
 If none of these options work for you, or you would like further clarification, please stop by our [gitter channel](https://gitter.im/m3db/Lobby) and we'll be happy to help you.
 
+## Performance configurations
```

Comment:
Might want to check with the db guys, this seems like it would be better suited in a general `Performance Tuning` section